security/security-design/fn-datasafe-dbaudit-to-oci-logging/terraform/modules/network/main.tf

```diff
@@ -0,0 +1,115 @@
+###############################################################################
+# Copyright (c) 2022, 2023, Oracle and/or its affiliates.  All rights reserved.
+# This software is dual-licensed to you under the Universal Permissive License
+# (UPL) 1.0 as shown at https://oss.oracle.com/licenses/upl.
+###############################################################################
+#
+# Author: Fabrizio Zarri
+#
+################################################################################
+
+
+resource "oci_core_virtual_network" "vcn" {
+  cidr_block     = var.VCN-CIDR
+  dns_label      = local.vcn_dns_label
+  compartment_id = var.compartment_ocid
+  display_name   = local.vcn_displayname
+}
+
+data "oci_core_services" "service_gateway_all_oci_services" {
+  filter {
+    name   = "name"
+    values = ["All .* Services In Oracle Services Network"]
+    regex  = true
+  }
+}
+
+resource "oci_core_service_gateway" "service_gw" {
+
+  compartment_id = var.compartment_ocid
+  vcn_id          = oci_core_virtual_network.vcn.id
+  services {
+        service_id = lookup(data.oci_core_services.service_gateway_all_oci_services.services[0], "id")
+  }
+  display_name = local.service_gw_displayname
+}
+
+resource "oci_core_route_table" "rt_fn_subnet" {
+  compartment_id = var.compartment_ocid
+  vcn_id         = oci_core_virtual_network.vcn.id
+  display_name   = local.vcnnameroutingtable_displayname
```

Comment:
Terraform naming convention is all lower case and words separated by '_'.

Reply:
I have update the name of all variables and references. Now are only present lower case and words separated by '_'.